status/barcode.py

```diff
@@ -49,7 +49,10 @@ def post(self):
                         print_barcode(linesToPrint)
                 else: # file submitted is a text file
                     for line in linesToPrint.splitlines(): # split into the different lines of the text file
-                        createdLabel = make_barcode(line, match_barcode(line, False))
+                        if self.get_argument('print_with_barcode', '') : # is true when barcode box is ticked
```

Comment:
If `print_with_barcode`  is boolean, why not just pass it to `make_barcode` like so
`make_barcode(line, match_barcode(line, self.get_argument('print_with_barcode', False))`

Reply:
It's fine the way it is as well